### PR TITLE
fix: cypress issues

### PR DESCRIPTION
**Pull request by karanh37**
### Describe your changes:

Fixes Cypress

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/common/serviceUtils.js [hunk 1]
```diff
@@ -29,7 +30,16 @@ export const visitServiceDetailsPage = (service, verifyHeader = true) => {
     .should('be.visible')
     .click();
 
-  verifyResponseStatusCode('@getServices', 200);
+  cy.wait('@getServices').then((interception) => {
+    const responseBody = interception.response.body;
+    cy.log(responseBody);
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/common/serviceUtils.js:35]: is it necessary to log in to the right panel? we can remove this

File: openmetadata-ui/src/main/resources/ui/cypress/constants/constants.js [hunk 2]
```diff
@@ -236,55 +236,55 @@ export const NEW_TAG = {
...
-  name: 'Business Glossary',
-  description: 'This is the Business glossary',
+  name: 'Cypress Glossary',
+  description: 'This is the Cypress Glossary',
   reviewer: 'Aaron Johnson',
   tag: 'PII.None',
 };
 export const NEW_GLOSSARY_1 = {
-  name: 'Product%Glossary',
+  name: 'Cypress%Glossary',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/constants/constants.js:245]: I think we should use 2 different name eg `Cypress Product%Glossary`

File: openmetadata-ui/src/main/resources/ui/cypress/constants/constants.js [hunk 3]
```diff
@@ -236,55 +236,55 @@ export const NEW_TAG = {
...
+  fullyQualifiedName: 'Cypress Glossary.Accounts',
 };
 
 export const NEW_GLOSSARY_1_TERMS = {
   term_1: {
     name: 'Features%Term',
     description: 'This is the Features',
     synonyms: 'data,collect,time',
-    fullyQualifiedName: 'Product%Glossary.Features%Term',
+    fullyQualifiedName: 'Cypress%Glossary.Features%Term',
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/constants/constants.js:281]: if we update name, we need to update fqn

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 4]
```diff
@@ -683,7 +727,7 @@ describe('Glossary page should work properly', () => {
     interceptURL('PATCH', '/api/v1/tables/*', 'saveTag');
 
     cy.get('[data-testid="saveAssociatedTag"]').scrollIntoView();
-    cy.get('[data-testid="saveAssociatedTag"]').click();
+    cy.get('[data-testid="saveAssociatedTag"]').click({ force: true });
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:730]: why do we need a force click?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 5]
```diff
@@ -702,7 +746,9 @@ describe('Glossary page should work properly', () => {
     // Select 2nd term
     cy.get('[data-testid="tag-selector"]').click().type(term4);
     cy.get(`[data-testid="tag-${glossary1}.${term4}"]`).click();
-    cy.get('[data-testid="saveAssociatedTag"]').scrollIntoView().click();
+    cy.get('[data-testid="saveAssociatedTag"]')
+      .scrollIntoView()
+      .click({ force: true });
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:751]: force click?

**Review by ShaileshParmar11 — COMMENTED** [review]
left few comment

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**